trio/_tests/test_exports.py

```diff
@@ -494,9 +494,25 @@ def lookup_symbol(symbol):
                 missing.remove("__aiter__")
                 missing.remove("__anext__")
 
-        # intentionally hidden behind type guard
+            # __getattr__ is intentionally hidden behind type guard. That hook then
+            # forwards property accesses to PurePath, meaning these names aren't directly on
+            # the class.
         if class_ == trio.Path:
```

Comment:
Is the indentation above a bit wonky or is that just GitHub's diff?

Reply:
Yep, bad indentation.